### PR TITLE
Add codemods to upgrade guide

### PR DESCRIPTION
**Pull request by rickhanlonii**
Adds the react-codemod codemods added by @alexbit-codemod and the team from [codemod.com](https://codemod.com).

### CODE REVIEW

File: src/content/blog/2024/04/25/react-19-upgrade-guide.md [hunk 1]
```diff
@@ -97,6 +98,36 @@ If you're using TypeScript, you also need to update the types. Once React 19 is
 
 We're also including a codemod for the most common replacements. See [TypeScript changes](#typescript-changes) below.
 
+## Codemods {/*codemods*/}
+
+To help with the upgrade, we've worked with the team at [codemod.com](https://codemod.com) to publish codemods that will automatically update your code to many of the new APIs and patterns in React 19.
+
+The Codemod team have published all codemods to [`react-codemod`](https://github.com/reactjs/react-codemod) and will be helping maintain the React codemods moving forward, but we recommend using the `codemod` command for a better experience.
```
Comment (poteto) [inline src/content/blog/2024/04/25/react-19-upgrade-guide.md:105]: this sentence is a little hard to parse for me, what does it mean to use the `codemod` command (as opposed to?)

**Review by poteto — APPROVED** [review]

File: src/content/blog/2024/04/25/react-19-upgrade-guide.md (hunk 1 above)
Comment (rickhanlonii) [reply src/content/blog/2024/04/25/react-19-upgrade-guide.md:105]: Will update with a better blurb

File: src/content/blog/2024/04/25/react-19-upgrade-guide.md [hunk 2]
```diff
@@ -97,6 +98,36 @@ If you're using TypeScript, you also need to update the types. Once React 19 is
...
+- [`replace-use-form-state`](https://github.com/reactjs/react-codemod?tab=readme-ov-file#replace-use-form-state) 
+- [`prop-types-typescript`](TODO)
+
+This does not include the TypeScript changes. See [TypeScript changes](#typescript-changes) below.
+
+</Note>
+
+Changes that include a codemod include the command below. 
+
+For a list of all available codemods, see the `react-codemod` [repo here](https://github.com/reactjs/react-codemod).
```
Comment (AhmedBaset) [inline src/content/blog/2024/04/25/react-19-upgrade-guide.md:130]:
```suggestion
For a list of all available codemods, see the [`react-codemod` repo](https://github.com/reactjs/react-codemod).
```
